### ISSUE TITLE
Create view for blog post page

### ISSUE THREAD

**Issue by pmcghen** [OutdoorFunNetwork/website#23]
See [FMS #4](https://github.com/OutdoorFunNetwork/fms/issues/4) for model information. 

The blog post should include a header that provides the post title, date, category, and author information. Author name should link to bio.

The router should be set up using the following URL pattern: `/blog/{category}/{yyyy}/{mm}/{dd}/{slug}`

**Comment by eric-miller2129** [comment]
I disagree with the slug format, one it's Hella long, but also posts can have multiple categories so we shouldn't restrict the slug to just one.

Also I have the slugs set up to be unique if that's what you were trying to fix with the date, but if not we can leave the date.

**Comment by pmcghen** [comment]
Yeah, if the posts can have more than 1 category I agree with removing that. The date was partially there to keep it unique, partially just preference on my part to have another place to show how old a post is.
